drivers/sensor/pmw3360/pmw3360.c

```diff
@@ -266,17 +265,14 @@ static int reg_read(struct pmw3360_data *dev_data, uint8_t reg, uint8_t *buf)
 	return 0;
 }
 
-static int reg_write(struct pmw3360_data *dev_data, uint8_t reg, uint8_t val)
+static int reg_write(const struct device *dev, uint8_t reg, uint8_t val)
 {
 	int err;
+	struct pmw3360_data *dev_data = dev->data;
+	const struct pmw3360_config *dev_config = dev->config;
 
 	__ASSERT_NO_MSG((reg & SPI_WRITE_BIT) == 0);
 
-	err = spi_cs_ctrl(dev_data, true);
-	if (err) {
-		return err;
-	}
-
```

Comment:
This should not be removed.

Reply:
oops missed re-adding those for write, fixed